src/quo2/components/wallet_user_avatar.cljs

```diff
@@ -0,0 +1,95 @@
+                     :brown {:text-color       colors/brown-opa-50
+                             :background-color colors/brown-opa-20}
+                     :beige {:text-color       colors/beige-opa-50
+                             :background-color colors/beige-opa-20}}
+             :dark  {:primary   {:text-color       colors/primary-60
+                                 :background-color colors/primary-20}
+                     :purple {:text-color       colors/purple-opa-60
+                              :background-color colors/purple-opa-20}
+                     :indigo {:text-color       colors/indigo-opa-60
+                              :background-color colors/indigo-opa-20}
+                     :turquoise {:text-color       colors/turquoise-opa-60
+                                 :background-color colors/turquoise-opa-20}
+                     :blue {:text-color       colors/blue-opa-60
+                            :background-color colors/blue-opa-20}
+                     :green {:text-color       colors/green-opa-60
+                             :background-color colors/green-opa-20}
+                     :yellow {:text-color       colors/yellow-opa-60
+                              :background-color colors/yellow-opa-20}
+                     :orange {:text-color       colors/orange-opa-60
+                              :background-color colors/orange-opa-20}
+                     :red {:text-color       colors/red-opa-60
+                           :background-color colors/red-opa-20}
+                     :pink {:text-color       colors/pink-opa-60
+                            :background-color colors/pink-opa-20}
+                     :brown {:text-color       colors/brown-opa-60
+                             :background-color colors/brown-opa-20}
+                     :beige {:text-color       colors/beige-opa-60
+                             :background-color colors/beige-opa-20}}})
+
+(def circle-sizes {:small 20
+                   :medium 32
+                   :large 48
+                   :x-large 80})
+
+(def font-sizes {:x-large 27
+                 :large 15
+                 :medium 13
+                 :small 11})
+
+(defn wallet-user-avatar
+  "params, first name, last name, color, size
+   and if it's dark or not!"
+  [{:keys [f-name l-name color dark? size] :or {f-name "John"
+                                                l-name "Doe"
+                                                color :red
+                                                dark? false
+                                                size :x-large}}]
+  (let [circle-size (size circle-sizes)
+        small? (= size :small)
+        f-name-initial (-> f-name
+                           clojure-string/upper-case
+                           (subs 0 1))
+        l-name-initial (-> l-name
+                           clojure-string/upper-case
+                           (subs 0 1))
+        is-theme-dark? (if dark? :dark :light)
```

Comment:
`is-theme-dark?` more looks like `theme`

Reply:
Renamed